lms/static/scripts/frontend_apps/components/dashboard/OrganizationActivityFilters.tsx

```diff
@@ -0,0 +1,114 @@
+import { MultiSelect } from '@hypothesis/frontend-shared';
+
+import type { Assignment, Course, Student } from '../../api-types';
+import { useConfig } from '../../config';
+import { useAPIFetch } from '../../utils/api';
+
+export type DashboardFiltersProps = {
+  selectedCourses: Course[];
+  onCoursesChange: (newCourses: Course[]) => void;
+  selectedAssignments: Assignment[];
+  onAssignmentsChange: (newAssignments: Assignment[]) => void;
+  selectedStudents: Student[];
+  onStudentsChange: (newStudents: Student[]) => void;
+};
+
+export default function OrganizationActivityFilters({
```

Comment:
I expect there would be quite a bit of duplication if you created a similar component for each view. However it isn't clear to me yet what the final set of distinct components will end up being.

Reply:
Yeah. I started up trying to re-use one component, then somehow came to the conclusion it would be too confusing and moved to this approach.

But now, as I have started to work on the assignments section filters, I'm again starting to lean towards reusing one component. I might just rename this component to something more generic and re-use it, but that shouldn't prevent us from merging this as is.